styles/header.css

```diff
@@ -0,0 +1,169 @@
+/* ================ Header ================ */
+.header.scroll-header {
+  background-color: var(--body-color);
+}
+
+.header.scroll-header .nav__logo {
+  color: var(--dark-color);
+}
+
+.header.scroll-header .nav_btns a {
+  color: var(--dark-color);
+  cursor: pointer;
+}
+
+.header.scroll-header .nav_btns .logout {
+  color: var(--dark-color);
+}
+
+.navbar {
+  padding: var(--sp-0-5) var(--sp-1);
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+}
+
+.navbar .nav_btns a {
+  margin: 0 var(--sp-0-5);
+  color: var(--white-color);
+  font-size: var(--h2-font-size);
+  cursor: pointer;
+}
+
+nav {
+  position: fixed;
+  top: 0;
+  right: 0;
+  height: 100vh;
+  width: 70%;
+  padding: 0.25rem;
+  text-transform: uppercase;
+  background-color: white;
+  z-index: var(--z-fixed);
+  -webkit-transform: translateX(100%);
+          transform: translateX(100%);
+  -webkit-transition: -webkit-transform 0.4s ease-in-out;
+  transition: -webkit-transform 0.4s ease-in-out;
+  transition: transform 0.4s ease-in-out;
+  transition: transform 0.4s ease-in-out, -webkit-transform 0.4s ease-in-out;
+}
+
+nav .nav_menu {
+  margin-top: var(--sp-4);
+}
+
+nav .nav_menu .nav__link {
+  color: #2d3748;
+  padding: var(--sp-1);
+  display: block;
+}
+
+nav .nav_menu .nav__link:hover {
+  background-color: #cbd5e1;
+}
```

Comment:
These **colors** also can be declared as `css variables` .

Reply:
Thanks, I forgot to apply the variables here after declaring global variables. I will do that.